pikelet/src/lang/surface/lexer.rs

```diff
@@ -1,13 +1,14 @@
 use logos::Logos;
 use std::fmt;
-use std::ops::Range;
+
+use crate::reporting::LexerError;
 
 /// Tokens in the surface language.
 #[derive(Debug, Clone, Logos)]
 pub enum Token<'a> {
     #[regex(r"\|\|\|(.*)\n", |lexer| lexer.slice())]
     DocComment(&'a str),
-    #[regex(r#"('[^'\\]|\\')*'"#, |lexer| lexer.slice())]
+    #[regex(r#"'([^'\\]|\\')*'"#, |lexer| lexer.slice())]
```

Comment:
Random thing I noticed (but unrelated to this patch), I believe that these callbacks now default to `|lexer| lexer.slice()`, so these could be left out.

Reply:
ohhhh nice!